net/packet/packet.go

```diff
@@ -424,6 +434,40 @@ func (q *Parsed) IsEchoResponse() bool {
 	}
 }
 
+// RemoveECNBits modifies p and its underlying memory buffer to remove
+// ECN bits, if any. It reports whether it did so.
+//
+// It currently only does the TCP flags.
+func (p *Parsed) RemoveECNBits() bool {
```

Comment:
Nit: none of the callers seem to care about the return value. Remove it? But it's also fine as-is.

Reply:
test uses it! :)

I figured it might be useful in the future to decide when you need to deep copy the result or something, knowing the underlying buffer has been mutated.